modules/core/src/weaver/Expectations.scala

```diff
@@ -150,6 +150,8 @@ object Expectations {
     }
 
     implicit class StringOps(str: String) {
+      def ignore(implicit loc: SourceLocation): TestName =
```

Comment:
IMO `ignored` is more consistent with `tagged`

Reply:
We'll keep `ignore` on the account that it's consistent with Munit : https://scalameta.org/munit/docs/filtering.html#ignore-single-test-case